server/routes/index.coffee

```diff
@@ -135,7 +135,7 @@ module.exports.setup = (app) ->
   app.put('/db/clan/:handle/remove/:memberHandle', mw.auth.checkLoggedIn(), mw.clans.removeMember)
 
   app.post('/db/classroom', mw.classrooms.post)
-  app.get('/db/classroom', mw.classrooms.fetchByCode, mw.classrooms.getByOwner)
+  app.get('/db/classroom', mw.classrooms.fetchByCode, mw.classrooms.getByOwner, mw.classrooms.getByMember)
```

Comment:
Can we rename/update `fetchByCode` to be `getByCode` and use the same `next()` check pattern?
This would make this routing waterfall more clear, instead of wondering if that first handler is fundamentally different.

Reply:
👍 It's already using the `next` pattern, can totally rename.